.circleci/config.yml

```diff
@@ -0,0 +1,118 @@
+version: 2.1
+jobs:
+    build-recent:
+        docker:
+          - image: cimg/base:stable
+            auth:
+                username: mydockerhub-user
+                password: $DOCKERHUB_PASSWORD
+            environment:
+                CMAKE_BUILD_TYPE: Release
+                FLAGS: -Wall -Werror
+                INSTALL: ~/osmt-install
+                USE_READLINE: OFF
+
+        steps:
+            - checkout
+            - run:
+                name: apt-get update
+                command: sudo apt-get update
+            - run:
+                name: install dependencies
+                command: sudo apt-get install libgmp-dev libedit-dev bison flex cmake libubsan0
+            - run:
+                name: Create the build directory
+                command: mkdir build
+            - run:
+                name: Release build gcc
+                command: ./ci/run_travis_commands.sh
+                environment:
+                    CMAKE_BUILD_TYPE: Release
+            - run:
+                name: Debug build gcc
+                command: ./ci/run_travis_commands.sh
+                environment:
+                    CMAKE_BUILD_TYPE: Debug
+            - run:
+                name: Clean build directory
+                command: |
+                    rm -rf build
+                    mkdir build
+            - run:
+                name: Install clang
+                command: |
+                    sudo apt-get install clang
+                    sudo update-alternatives --install /usr/bin/c++ c++ /usr/bin/clang++ 60
```

Comment:
This changes the C++ compiler to point to clang so that the next commands will compile with clang?

Reply:
Correct.  debian uses this update-alternatives system to support several compilers.